docs/index.md

```diff
@@ -45,42 +63,88 @@ class CounterRenderView(root: ViewGroup): RenderView<CounterRenderModel> {
 }
 ```
 
-We now defined a single entry-point to our rendering (this makes debugging issues a lot easier). Anytime 
-you need to update UI, just set a new Render Model.
+This is createing a `Renderer` that has a TextView for displaying the current count value state along with a
+increment and decrement button with click listeners that call the `onDecrement()` and `onIncrement()` methods
+you created in your `CounterRenderModel`. This codce also references some view elements that you don't 
+currently have. To fix that replace your activity view with the following.
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I wonder if we should include this whole XML, makes the example pretty dense. Maybe it would be enough to include a screenshot of the UI and provide a link to XML layout.

Reply:
If we want to have an example that a user can follow along with, unless we have a starter project, this is the only other way to do that given that these are referenced in our examples.